iBudget/spending/views.py

```diff
@@ -74,3 +74,77 @@ def set_spending_limitation_ind(request):
         spending_limitation_ind.save()
 
     return HttpResponse(status=201)
+
+
+def group_limit(request):
+    """the functions finds all the shared spendings associated with particular user and
+    returns them
+    :param request object
+    """
+    if request.method == 'GET':
+        user_id = request.user
+        available_spendings = \
+        SpendingCategories.objects.filter(sharedspendingcategories__group__usersingroups__user_id=
```

Comment:
Could you please explain why you select only SpendingCategories for the users that are group admin?

Reply:
That's because according to our logic only group admins can set limits for group spendings, not all the group members.